pom.xml

```diff
@@ -245,11 +245,24 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-test</artifactId>
+            <version>4.2.1.RELEASE</version>
```

Comment:
You can remove the version tag because spring boot brings the dependency and the current version `4.2.4.RELEASE`